src/components/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+/* 
+  FOR PRIMARY BUTTON USAGE:
+
+    Props: 
+      primaryColor (Required) - This color will determine the default background color of the button
+      secondaryColor (Required) - This color will determine the border color and background color on hover
+   
+    Example:
+      <Button primaryColor={COLORS.primaryBlue} secondaryColor={COLORS.primaryBlueClicked} onClick={...}>
+        [Button text here]
+      </Button>
+
+  FOR SECONDARY BUTTON USAGE:
+
+    Props:
+      primaryColor (Omit) - IMPORTANT: DO NOT PASS IN A primaryColor PROP!!! The background color will be white by default
+      secondaryColor (Required) - This color will determine the border color and background color on hover
+
+    Example: 
+      <Button secondaryColor={COLORS.primaryBlue} onClick={...}>
+        [Button text here]
+      </Button>
+ */
+const Button = styled.button<{
+  primaryColor?: string;
+  secondaryColor: string;
+}>`
+  appearance: none;
+  color: ${props => (props.primaryColor ? 'white' : 'black')};
+  background: ${props => (props.primaryColor ? props.primaryColor : 'white')};
+  align-self: flex-end;
+  padding: 0.625rem 1.25rem;
+  border-radius: 5px;
```

Comment:
We may want to consider using `rem` units here

Reply:
fixed!